vignettes/safestats-vignette.Rmd

```diff
@@ -60,7 +60,7 @@ install.packages("safestats")
 The development version can be found on [GitHub](https://github.com/alexanderlynl/safestats), which can be installed with the `devtools` package from [CRAN](https://cran.r-project.org/package=devtools) by entering in `R`:
 
 ```{r devtools, eval=FALSE}
-devtools::install_github("AlexanderLyNL/safestats", build_vignettes = TRUE)
+devtools::install_github("AlexanderLyNL/safestats", ref = "logrank", build_vignettes = TRUE)
```

Comment:
Als logrank branch in dev gemerged is, is dit niet meer nodig toch?

Reply:
Ja goeie, ik maak wel een issue aan dan vergeet ik het niet. Wanneer het in development is, dan verander ik het wel weer terug.

Reply:
Fixes https://github.com/AlexanderLyNL/safestats/issues/22